### PR TITLE
[EN/FR] Update "Beatmap editor/Compose"; fix small mistake in "Beatmap editor/Beat snap divisor"

### PR DESCRIPTION
**Pull request by DoryanR**
Proposal to update the French translation of the wiki page "Compose".

- Addition of the French images for the article `Compose`

Also:

- In `Beat Snap Divisor/fr.md` and `en.md` : convert standard > osu!, and lowercase title in `fr.md`

- [x] Waiting for a recheck from @Satsukel on some points seen together

### CODE REVIEW

File: wiki/Beatmap_Editor/Compose/fr.md [hunk 1]
```diff
@@ -1,170 +1,166 @@
...
+| Spinner (`4`) | **Clic gauche/droit :** Début/Fin du Spinner à l'horodatage actuel. | Spinner. Veillez à ce que l'Auto puisse obtenir au moins 2000 bonus par spinner (pour éviter le fameux spinner spam/ninja spinner). |
 
-### Centre (Terrain de jeu)
+### Au centre (Terrain de jeu)
 
-**Représentation visuelle de la cartographie basée sur l'horodatage actuel** Mettez votre design en carte ici et votre design apparaîtra exactement comme vous l'avez mis en carte pendant le jeu. Il s'agit d'une cartographie WYSIWYG (ce que vous voyez est ce que vous obtenez). Pour un mappage basé sur du texte (généralement pour un réglage fin), vous pouvez ouvrir le fichier `.osu` directement en utilisant le Bloc-notes.
+**Représentation visuelle du mappage basée sur l'horodatage actuel** Mappez votre design ici et ol apparaîtra exactement comme vous l'avez mappé pendant le jeu. Il s'agit d'un mappage WYSIWYG (what you see is what you get > ce que vous voyez est ce que vous obtenez). Pour un mapping en texte (généralement pour un réglage fin), vous pouvez ouvrir le fichier `.osu` directement en utilisant Notepad.
```
Comment (Satsukel) [inline wiki/Beatmap_Editor/Compose/fr.md:112]: Coquille
```suggestion
**Représentation visuelle du mappage basée sur l'horodatage actuel** Mappez votre design ici et il apparaîtra exactement comme vous l'avez mappé pendant le jeu. Il s'agit d'un mappage WYSIWYG (what you see is what you get > ce que vous voyez est ce que vous obtenez). Pour un mapping en texte (généralement pour un réglage fin), vous pouvez ouvrir le fichier `.osu` directement en utilisant Notepad.
```

**Review by Satsukel — CHANGES_REQUESTED** [review]
Correction de certaines fautes/coquilles

**Review by Satsukel — APPROVED** [review]

File: wiki/Beatmap_Editor/Compose/fr.md [hunk 2]
```diff
@@ -1,170 +1,166 @@
...
+**Liste des sous-outils disponibles en complément de la boîte à outils originale.**
 
-![Hitsounds & Assist Tools](img/BE_HSAT.jpg "Hitsounds et outils d'assistance")
+![Hitsounds & Outils d'assistance](img/BE_HSAT-FR.png "Hitsounds & Outils d'assistance")
 
-| Bouton (Raccourci clavier) | Utilisation | Description |
+| Bouton (raccourci clavier) | Utilisation | Description |
 | :-- | :-- | :-- |
-| Nouveau Combo (`Q`) | **Clic droit :** Changer la note courante à la couleur du combo suivant. | Combo de notes. C'est l'équivalent d'un couplet/d'une mesure dans le script d'une chanson/d'un instrument. Une plage acceptable est d'environ 5-20. |
+| Nouveau combo (`Q`) | **Clic droit :** Changez la note actuelle pour la couleur combinée suivante. | Note du combo. C'est l'équivalent d'un couplet/une mesure dans une chanson/un instrument. Une fourchette acceptable est d'environ 5 à 20. |
```
Comment (Satsukel) [inline wiki/Beatmap_Editor/Compose/fr.md:122]:
```suggestion
| Nouveau combo (`Q`) | **Clic droit :** Change la note actuelle en la couleur du combo suivant. | Note du combo. C'est l'équivalent d'un couplet/une mesure dans une chanson/un instrument. Une fourchette acceptable est d'environ 5 à 20. |
```

File: wiki/Beatmap_Editor/Compose/fr.md [hunk 3]
```diff
@@ -1,170 +1,166 @@
...
 
 #### Outils d'assistance
 
-| Bouton (Raccourci clavier) | Utilisation | Description |
+| Bouton (raccourci clavier) | Utilisation | Description |
 | :-- | :-- | :-- |
-| Grid Snap (`T`) | **`Shift` (Hold):** Basculement temporaire. **`Ctrl` (Hold):** Désactivation temporaire de l'accrochage de l'angle. | Par défaut, les notes frappées sont accrochées à la grille. |
-| Distance Snap (`Y`) | **`Alt` (Hold):** Basculement temporaire. Basculer le diviseur Beat Snap sur le Distance Snap **`Alt` + Molette de la souris:** Ajuster le multiplicateur du Distance Snap. | La distance entre deux notes consécutives est calculée en fonction de leur différence rythmique. Il est préférable de l'utiliser lorsque la timeline est en pause. |
-| Verrouiller les notes (`L`) | ("Select" sur une note de Hit) **Clic gauche :** Verrouiller cette note | Verrouiller la note sélectionnée à la position et à l'horodatage actuels. |
+| Grid Snap (`T`) | **`Shift` (Maintenir):** On/Off temporaire. **`Ctrl` (Maintenir):** Désactiver temporairement l'angle du snapping. | Par défaut, les notes sont fixées à la grid. |
```
Comment (Satsukel) [inline wiki/Beatmap_Editor/Compose/fr.md:136]:
```suggestion
| Grid Snap (`T`) | **`Shift` (Maintenu):** On/Off temporaire. **`Ctrl` (Maintenu):** Désactiver temporairement l'angle du snapping. | Par défaut, les notes sont fixées à la grid. |
```

File: wiki/Beatmap_Editor/Compose/fr.md [hunk 4]
```diff
@@ -1,170 +1,166 @@
...
+| Distance Snap (`Y`) | **`Alt` (Maintenir):** On/Off temporaire. Bascule Beat Snap Divisor en Distance Snap **`Alt` + Molette de souris :** Ajustez le multiplicateur de distance snap | La distance entre des notes consécutives sera cassée en fonction de leur différence rythmique. A utiliser de préférence lorsque la timeline est en pause. |
```
Comment (Satsukel) [inline wiki/Beatmap_Editor/Compose/fr.md:137]:
```suggestion
| Distance Snap (`Y`) | **`Alt` (Maintenu):** On/Off temporaire. Bascule Beat Snap Divisor en Distance Snap **`Alt` + Molette de souris :** Ajustez le multiplicateur de distance snap | La distance entre des notes consécutives sera cassée en fonction de leur différence rythmique. A utiliser de préférence lorsque la timeline est en pause. |
```

File: wiki/Beatmap_Editor/Compose/fr.md [hunk 5]
```diff
@@ -1,170 +1,166 @@
...
+| Lock Notes (`L`) | ("Select" sur une note) **Clic gauche :** Verrouille cette note | Verrouille la note sélectionnée à la position et à l'horodatage actuels. |
```
Comment (Satsukel) [inline wiki/Beatmap_Editor/Compose/fr.md:138]:
```suggestion
| Lock Notes (`L`) | ("Select" sur une note) **Clic gauche :** Verrouiller cette note | Verrouille la note sélectionnée à la position et à l'horodatage actuels. |
```

File: wiki/Beatmap_Editor/Compose/fr.md [hunk 6]
```diff
@@ -173,183 +169,183 @@ Au **bas-centre**, il y a une **ligne de temps** avec des marquages et les bouto
...
+| Additions | Auto, Normal, Soft, Drum | Même chose que ci-dessus, MAIS les réglages des additions écrasent les réglages du sampleset. Exemple : Timing **Drum 55%**, Sampleset **Soft**, alors que cette Additions est **Normal**, jouera tous les sons **Normal** sur les notes comme clap, normal, finish, whistle avec un volume de 55%. |
+| Select (`1`) | **Clic gauche/glisser :** Déplacement de l'horodatage et de l'emplacement d'une note. **Clic droit :** Supprimer la note. **`Ctrl` + Clic :** Sélection multiple. | Sélectionnez et modifiez les notes existantes. |
+| Circle (`2`) | **Clic gauche/droit :** Placer/supprimer la note. | Ajoute une note à la colonne actuelle en fonction de la position de la souris. |
+| Hold (`3`) | **Clic gauche (maintien):** Placez une hold note et ajustez la longueur. **Relâcher à la fin:** Définir une longueur de hold | Ajoute une hold note à la colonne actuelle en fonction de la position de la souris. |
```
Comment (Satsukel) [inline wiki/Beatmap_Editor/Compose/fr.md:209]:
```suggestion
| Hold (`3`) | **Clic gauche (maintenu):** Place une hold note et ajuste la longueur. **Relâcher à la fin:** Définit la longueur de la hold note | Ajoute une hold note à la colonne actuelle en fonction de la position de la souris. |
```

File: wiki/Beatmap_Editor/Compose/fr.md [hunk 7]
```diff
@@ -173,183 +169,183 @@ Au **bas-centre**, il y a une **ligne de temps** avec des marquages et les bouto
...
-| Verrouiller les notes (`L`) | ("Select" sur une note de Hit) **Clic gauche :** Verrouiller cette note | Verrouiller la note sélectionnée à la position et à l'horodatage actuels. |
+| Grid Snap (`T`) | **`Shift` (Maintenir):** On/Off temporaire. **`Ctrl` (Maintenir):** Désactiver temporairement l'angle du snapping. | Par défaut, les notes sont fixées à la grille. |
+| Distance Snap (`Y`) | **`Alt` (Maintenir):** On/Off temporaire. Passe du Beat Snap Divisor à la Distance Snap **`Alt` + Molette de la souris :** Ajuster le multiplicateur de distance de Snap. | La distance entre des notes consécutives sera snappé en fonction de leur différence rythmique. A utiliser de préférence lorsque la timeline est en pause. |
+| Lock Notes (`L`) | ("Select" sur une note) **Clic gauche :** Verrouiller cette note | Verrouille la note sélectionnée à la position et à l'horodatage actuels. |
```
Comment (Satsukel) [inline wiki/Beatmap_Editor/Compose/fr.md:289]:
```suggestion
| Lock Notes (`L`) | ("Select" sur une note) **Clic gauche :** Verrouille cette note | Verrouille la note sélectionnée à la position et à l'horodatage actuels. |
```

File: wiki/Beatmap_Editor/Compose/fr.md [hunk 8]
```diff
@@ -1,170 +1,166 @@
...
+| Additions | Auto, Normal, Soft, Drum | L'ajout de hitsound signifie ajouter des Clap, des Finish, et des Whistle avec le set sélectionné. Exemple avec un Sampleset **Normal**, le Timing point sera **Soft 77%**, alors que Additions est réglé sur **Drum** et ont **Clap** sur les notes, le résultat sera normal-hitnormal 77% + drum-hitclap 77% sur les frappes |
+| Select (`1`) | **Clic gauche/glisser :** Sélectionnez/ajustez l'emplacement des notes/sliderpoints. **Clic droit :** Supprimez les notes/sliderpoints. **`Ctrl` + Clic :** Sélection multiple. | Sélectionnez et modifiez les notes existantes. |
+| Circle (`2`) | **Clic gauche/droit :** Ajout/suppression d'un cercle | Ajout d'un cercle |
+| Slider (`3`) | **Clic gauche/droit :** Début/Fin de slider. Ajouter/supprimer des sliderpoints (carré blanc). **Clic gauche x2 :** Nouvelle section de courbe (carré rouge). | Slider. |
```
Comment (Satsukel) [inline wiki/Beatmap_Editor/Compose/fr.md:107]:
```suggestion
| Slider (`3`) | **Clic gauche/droit :** Début/Fin de slider. Ajoute/supprime des sliderpoints (carré blanc). **Clic gauche x2 :** Nouvelle section de courbe (carré rouge). | Slider. |
```

**Review by Satsukel — CHANGES_REQUESTED** [review]

File: wiki/Beatmap_Editor/Compose/fr.md [hunk 9]
```diff
@@ -173,183 +169,183 @@ Au **bas-centre**, il y a une **ligne de temps** avec des marquages et les bouto
...
+| Additions | Auto, Normal, Soft, Drum | Même chose que ci-dessus, MAIS les réglages des additions écrasent les réglages du sampleset. Exemple : Timing **Drum 55%**, Sampleset **Soft**, alors que cette Additions est **Normal**, jouera tous les sons **Normal** sur les notes comme clap, normal, finish, whistle avec un volume de 55%. |
+| Select (`1`) | **Clic gauche/glisser :** Déplacement de l'horodatage et de l'emplacement d'une note. **Clic droit :** Supprimer la note. **`Ctrl` + Clic :** Sélection multiple. | Sélectionne et modifie les notes existantes. |
+| Circle (`2`) | **Clic gauche/droit :** Placer/supprimer la note. | Ajoute une note à la colonne actuelle en fonction de la position de la souris. |
+| Hold (`3`) | **Clic gauche (maintien):** Placez une hold note et ajustez la longueur. **Relâcher à la fin:** Définir une longueur de hold | Ajoute une hold note à la colonne actuelle en fonction de la position de la souris. |
```
Comment (DoryanR) [inline wiki/Beatmap_Editor/Compose/fr.md:209]:
```suggestion
| Hold (`3`) | **Clic gauche (maintenu) :** Place une hold note et ajuste la longueur. **Relâcher à la fin: ** Définit la longueur de la hold note | Ajoute une hold note à la colonne actuelle en fonction de la position de la souris. |

```

**Review by Satsukel — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**